app/controllers/api/v2/auth_source_externals_controller.rb

```diff
@@ -0,0 +1,48 @@
+module Api
+  module V2
+    class AuthSourceExternalsController < V2::BaseController
+      include Foreman::Controller::Parameters::AuthSourceExternal
+
+      before_action :find_resource, :only => %w{show update}
+
+      api :GET, "/auth_source_externals/", N_("List external authentication sources")
+      api :GET, '/locations/:location_id/auth_source_externals/',
+          N_('List external authentication sources per location')
+      api :GET, '/organizations/:organization_id/auth_source_externals/',
```

Comment:
this does not seem to work, seems the route for this endpoint is not defined, the same for location and most likely all auth source types, either remove it or add routes

Reply:
resolved now